packages/ui-reactive/src/BestNumber.tsx

```diff
@@ -28,7 +28,7 @@ class BestNumber extends React.PureComponent<Props> {
           chain_bestNumber
             ? formatNumber(chain_bestNumber)
             : '-'
-          }{children}
+        }{children}
```

Comment:
not sure why my cmd+s did this... though I would argue this one looks better

Reply:
And more correct. Been seeing some funky spacing not being complained about lately :(